sql/character.anticheat.sql

```diff
@@ -0,0 +1,30 @@
+DROP TABLE IF EXISTS `daily_players_reports`;
```

Comment:
you know, you can simply rename file instead delete + recreate xd

Reply:
at the moment, i only updated the sql to mysql 8.0 standard, it was not rewritten outside of just its standard being updated.

Reply:
@Aokromes do you need me to do a drop and create instead for both tables in the character.anticheat.sql?

Reply:
done via https://github.com/TrinityCore/TrinityCoreCustomChanges/pull/73/commits/375b8a851bad89b2bf6b6aa811150cf8126cfc6e